fulfillment-service/proto/private/osac/private/v1/baremetal_instance_types_service.proto

```
@@ -15,20 +15,60 @@ syntax = "proto3";
 
 package osac.private.v1;
 
+import "cleanapi/cleanapi.proto";
 import "google/api/annotations.proto";
 import "google/protobuf/field_mask.proto";
 import "osac/private/v1/baremetal_instance_type_type.proto";
 
+option (cleanapi.file).package = "osac.public.v1";
```

Buf (1.72.0)

[error] cannot find `cleanapi.file` in this scope

(COMPILE)